packages/cactus-plugin-ledger-connector-quorum/src/main/json/openapi.json

```diff
@@ -441,6 +441,114 @@
                     }
                 }
             },
+            "RunTransactionGethKeychainRequest": {
+                "type": "object",
+                "required": [
+                    "web3SigningCredential",
+                    "transactionConfig"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                    "web3SigningCredential": {
```

Comment:
@TonyRowntree You could narrow down the type of the signing credential here otherwise people could assume that they can use any one of the possible signing credential types.